lib/vtls/openssl.c

```diff
@@ -146,6 +146,20 @@ static unsigned long OpenSSL_version_num(void)
 #define OPENSSL_load_builtin_modules(x)
 #endif
 
+/*
+ * Whether SSL_CTX_set_keylog_callback is available.
+ * OpenSSL: supported since 1.1.1 https://github.com/openssl/openssl/pull/2287
+ * BoringSSL: supported since d28f59c27bac (committed 2015-11-19), the
+ *            BORINGSSL_201512 macro for 2016-01-21 should be close enough.
```

Comment:
Oh, I just noticed that this "for" should be a "from"

See https://github.com/curl/curl/pull/1346/commits/f083a44595e81837d1ba6058a53cb718053579f0